capstone-ui/src/Components/ResultsBox/ResultsBox.jsx

```diff
@@ -120,7 +120,7 @@ export default function ResultsBox( { locationLevel, countyFIPS, selectedCandida
                     onClick={() => handleClick(candidate.key, candidate.name, event)}
                     className={ (selectedCandidates && (selectedCandidates.some((c) => c.key === candidate.key))) ? "selected" : ""}
                     >
-                  <td>{candidate.name}</td>
+                  <td>{candidate.name != "None of these candidates" ? (candidate.name) : ("Independent")}</td>
```

Comment:
Lets put `None of these candidates` string literal into a constant variable and check that constant here. It is a lot cleaner and makes it easier to read.